content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,40 @@ menu:
 weight: 101
 ---
 
+## v2.0.3 General Availability [2020-12-08]
+
+### Breaking Changes
+
+Previously, `influxd upgrade` would attempt to write upgraded `config.toml` files into the same directory as the source
+`influxdb.conf` file. If this failed, a warning would be logged and `config.toml` would write into the `HOME` directory.
```

Comment:
`config,toml` was written to the `home` directory of the user who launched the upgrade.  (this describes the previous behavior).